packages/trpc/server/routers/loggedInViewer/outOfOffice.handler.ts

```diff
@@ -192,6 +216,59 @@ export const outOfOfficeCreate = async ({ ctx, input }: TBookingRedirect) => {
     }
   }
 
+  // Send webhook to notify other services
+  const subscriberOptions: GetSubscriberOptions = {
+    userId: ctx.user.id,
```

Comment:
`teamId` is missing here. We need to make sure to get all OOO_created webhooks from all teams the user is a member of